go.sum

```diff
@@ -80,6 +80,8 @@ github.com/infostellarinc/go-stellarstation v0.3.0 h1:p2j69fiqTEbRkEtBYeXYtIPbFa
 github.com/infostellarinc/go-stellarstation v0.3.0/go.mod h1:ZUgU88DoZXjmZ2XrIyYfEWlnCV3sYWMkcsRUMvcwNUg=
```

Comment:
OOT: Why do older versions stick around here?

Reply:
I can't find the exact reason why they keep it, but it a behavior of go mod. Though it won't harm you, you can remove it by `go mod tidy` command.
I find a ticket for golang to automatically run go tidy when changing dependencies.